src/main/java/ca/on/oicr/gsi/dimsum/service/filtering/PendingState.java

```diff
@@ -338,7 +338,7 @@ protected Function<Case, Stream<Sample>> getFilteredSamples(MetricCategory reque
     throw new IllegalStateException("This gate does not apply to samples");
   }
 
-  private static class Helpers {
+  public static class Helpers {
```

Comment:
Please create a public `SampleUtils` class in the `util` package and move the necessary methods to that instead of making parts of this inner class public. This way, they're easier to find and reference